app/Domains/Vault/ManageVault/Services/CreateVault.php

```diff
@@ -113,4 +115,47 @@ private function populateDefaultContactImportantDateTypes(): void
             'can_be_deleted' => false,
         ]);
     }
+
+    private function populateMoodTrackingParameters(): void
+    {
+        (new CreateMoodTrackingParameter())->execute([
+            'account_id' => $this->data['account_id'],
+            'author_id' => $this->author->id,
+            'vault_id' => $this->vault->id,
+            'label' => trans('vault.settings_mood_tracking_parameters_awesome'),
```

Comment:
Translated label should not go into the database directly...

Reply:
fixed